WWDCScholars/ScholarsViewController.swift

```diff
@@ -100,6 +107,16 @@ internal final class ScholarsViewController: UIViewController {
         })
     }
     
+    internal func checkForIntroSeenYet(){
+        let userDefaults = UserDefaults.standard
+        if userDefaults.bool(forKey: "userHasSeen2017Intro") == true{
```

Comment:
Should this be 2018 or are we going to pretend the previous launch did happen? :P

Reply:
Lol, my bad. Yeah, 18